src/views/Stack/Card.tsx

```diff
@@ -253,12 +253,7 @@ export default class Card extends React.Component<Props> {
   );
   private extrapolatedPosition = add(
     this.gesture,
-    multiply(
-      this.velocity,
-      this.velocity,
-      this.velocitySignum,
-      SWIPE_VELOCITY_IMPACT
-    )
+    multiply(this.velocity, this.velocitySignum, SWIPE_VELOCITY_IMPACT)
```

Comment:
According to @osdnk's changes in tabview, this can be changed to:

```js
private extrapolatedPosition = add(
  this.gesture,
  multiply(this.velocity, SWIPE_VELOCITY_IMPACT)
);
```

Reply:
Yes definitely. Please change in following way, @christianbaroni

Reply:
`this.velocitySignum` shouldn't be needed. I used bc this.velocity* this.velocity is always positive.